app/src/lib/stores/helpers/background-fetcher.ts

```diff
@@ -96,7 +96,10 @@ export class BackgroundFetcher {
       try {
         await this.fetch(this.repository)
       } catch (e) {
-        log.error('Error performing periodic fetch', e)
+        log.error(
+          `Error performing periodic fetch for '${this.repository.name}'`,
```

Comment:
Did we want to differentiate here between `owner/name` repositories and `folder` repositories, if `githubRepository` is defined?